src/dlls/mscoree/unixinterface.cpp

```diff
@@ -178,13 +178,13 @@ static void ExtractStartupFlagsAndConvertToUnicode(
 //
 extern "C"
 int coreclr_initialize(
-            const char* exePath,
```

Comment:
These indents were intentional to not to align the parameters with the name of the function / function code lines. Could you please revert this change?